x-pack/plugins/features/server/routes/index.test.ts

```diff
@@ -14,7 +14,7 @@ import { RequestHandler } from '../../../../../src/core/server';
 let currentLicenseLevel: string = 'gold';
 
 describe('GET /api/features', () => {
-  let routeHandler: RequestHandler<any, any, any>;
+  let routeHandler: RequestHandler<any, any, any, any>;
```

Comment:
the necessity to declare Method in generic type creates additional work for the NP plugins. what benefits did they get in return?